module/Calculator/ColorJudge.h

```diff
@@ -0,0 +1,75 @@
+/**
+ * @file ColorJudge.h
+ * @brief 色識別クラス
+ * @author keiya121
+ */
+
+#ifndef COLOR_JUDGE_H
+#define COLOR_JUDGE_H
+
+#include <string.h>
+#include <algorithm>
+#include "Measurer.h"
+
+enum class COLOR : int {
+  NONE = 0,
+  BLACK = 1,
+  WHITE = 2,
+  BLUE = 3,
+  GREEN = 4,
+  YELLOW = 5,
+  RED = 6,
+};
+
+struct Hsv {
+  int hue;         // 色相(0-360)
+  int saturation;  // 彩度(0-100)
+  int value;       // 明度(0-255)
+};
+
+class ColorJudge {
+ public:
+  ColorJudge() = delete;  // 明示的にインスタンス化を禁止
+
+  /**
+   * 与えられたRGB値から、何色かを返す
+   * @param rgb RGB
```

Comment:
下記の方が、一目でどっちが引数名か分かりやすいかな。
```suggestion
   * @param rgb RGB値
```